tools/mcsema_disass/ida/get_cfg.py

```diff
@@ -1324,12 +1328,13 @@ def identify_external_symbols():
             extern_name = found_name
             break
 
-        DEBUG("WARNING: Adding variable {} at {:x} as external".format(
-            extern_name, ea))
+        if extern_name != "__gxx_personality_v0":
```

Comment:
This should be a global variable and a set, in case more exceptions must be made.